doc/source/notebooks/basic.ipynb

```diff
@@ -1,14 +1,28 @@
 {
     "cells": [
         {
+            "attachments": {},
             "cell_type": "markdown",
-            "metadata": {
-                "tags": []
-            },
+            "metadata": {},
             "source": [
-                "### Prepare Data\n",
+                "### Prerequisites\n",
+                "\n",
+                "To run this tutorial, we need\n",
+                "\n",
+                "- A database that is authorized to access, and\n",
+                "- A table named `demo` as created with the following SQL command:\n",
+                "\n",
+                "    ```sql\n",
+                "    CREATE TABLE demo AS\n",
+                "    SELECT n AS i, n AS j, n AS k\n",
+                "    FROM generate_series(0,9) AS n;\n",
+                "    ```\n",
                 "\n",
-                "First, connect to the `gpadmin` database and populate some data into `demo` table."
+                "To create this table, if in a shell environment, [`psql`](https://www.postgresql.org/docs/current/app-psql.html) can be used.\n",
+                "\n",
+                "Or, inside a Jupyter Notebook, the SQL command can be executed directly in cells with [ipython-sql](https://pypi.org/project/ipython-sql/), as shown below.\n",
```

Comment:
Kind of too detailed. To make the doc simple, I think we can skip the details steps of how to create the demo table. Just give the table definition and assume user can create it in any way.